### ISSUE TITLE
Wifi connected, but no internet (It works before) (Monterey 12.2)

### ISSUE THREAD

**Issue by penguin-io** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#218]
Hey @chris1111!
TL:DR, so I have been using your wifi app from Catalina to Monterey, and it was working perfectly fine! (Of course, with my TL-WN725N Nano)
But later today, I can't get my internet access, even though the wifi is connected.
Maybe you can help me to solve this problem!
Looking forward to your reply.
Thanks!

P.S. : Here's some screenshots that might be helpful
![Screen Shot 2022-03-29 at 12 10 47](https://user-images.githubusercontent.com/54567544/160537705-4c5d96c4-ff18-4d35-a777-41c619f78a70.png)
![Screen Shot 2022-03-29 at 12 12 07](https://user-images.githubusercontent.com/54567544/160537991-24a8f0ba-1b67-40dd-9c37-42da4592ff17.png)

**Comment by chris1111** [comment]
from Catalina to Monterey?
if you have update Catalina to Monterey 12 of course you need uninstall, reboot then install again

**Comment by penguin-io** [comment]
Hey there @chris1111!
Yeah, I have already reinstall the wifi app at my first copy of Monterey, it's working fine yesterday. Today, the wifi status said connected, but doesn't an internet access. (Have already tried on my phone, It should have an internet access).

**Comment by chris1111** [comment]
SSBR, SBR 2 how much SBR you have ?

**Comment by penguin-io** [comment]
I have 2 SBR. Purposely made for each floor actually.

**Comment by chris1111** [comment]
This is trange because I have 6 Adapter for testing and I never have disconection even after updating macOS since macOS Big sur 11 Beta 1

**Comment by chris1111** [comment]
Thats not append on my 6 adapter.
This is as no relation with Wireless-USB-OC-Big-Sur-Adapter, its relate to your side
Issue closed

**Comment by penguin-io** [comment]
Hey there @chris1111!
After looking for several hours of research, found out that it was DHCP router problem.
Thanks @chris1111!
